OracleDataIntegrator/README.md

```diff
@@ -0,0 +1,138 @@
+
+         $ docker run --name ODI122126Database  -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/ODIVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
+
+The above command starts a DB container mounting a host directory as /opt/oracle/oradata for persistence. 
+It maps the containers 1521 and 5500 port to respective host port such that the services can be accessible outside of localhost.
+
+## ODI Distributable version 12.2.1.2.6 Docker image Creation and Running
+
+To build a ODI image either you can start from building Oracle JDK or use the already available Oracle JDK image.
+The Oracle JDK  image is not currently available.
+If plan to use available Oracle JDK image, skip next steps and jump to "Building Docker Image for ODI"
+
+### Building Oracle JDK (Server JRE) base image
+
+Oracle Server JRE image can be pulled from the [Docker Store](https://store.docker.com/images/oracle-serverjre-8) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleJava/java-8). For more information, visit the [OracleJava](../OracleJava) folder's [README](../OracleJava/README.md) file.
+
+### Building Docker Image for ODI
+
+**IMPORTANT:** you have to download the binary of ODI and put it in place (see `.download` files inside dockerfiles/<version>).
+
+
+To try a sample of a ODI image with a domain configured, follow the steps below:
+
+  1. Make sure you have **oracle/odi:12.2.1.2.6** image built. If not go into **dockerfiles** and call 
+
+        $ sh buildDockerImage.sh -v 12.2.1.2.6
+
+  2. Verify you now have the image **oracle/odi:12.2.1.2.6** in place with 
+
+        $ docker images
+
+**IMPORTANT:** the resulting images will NOT have a domain pre-configured. But, it has the scripts to create and configure a odi domain while creating a container out of the image.
+
+
+### Creating ODI container
+
+Create an environment file **odi.env.list**
+
+        CONNECTION_STRING=<Database Host Name>:<port#>/<ORACLE_PDB>
+        RCUPREFIX=<RCU_Prefix>
+        DB_PASSWORD=<database_password>
+        DB_SCHEMA_PASSWORD=<RCU schema Password>
+        SUPERVISOR_PASSWORD=<Password for SUPERVISOR>
+        WORK_REPO_NAME=<Name for WORK repository>
+        WORK_REPO_PASSWORD=<Password for WORK repository>
+        HOST_NAME=<Hostname where docker is running>
+        
+        
+Sample Data will look like this...
+
+        CONNECTION_STRING=<Database Host Name>:1521/odipdb
+        RCUPREFIX=ODI1
+        DB_PASSWORD=Welcome1
+        DB_SCHEMA_PASSWORD=Welcome1
+        SUPERVISOR_PASSWORD=Welcome1
+        WORK_REPO_NAME=WORKREP
+        WORK_REPO_PASSWORD=Welcome1
+        HOST_NAME=<Hostname where ODI docker container is running>
+
+To start a docker container with a ODI domain and Agent, call docker run command and pass the above odi.env.list file.
```

Comment:
Rewrite to: "To start a container with an ODI domain and agent, run the following command:"